pkgs/games/openxray/default.nix

```diff
@@ -66,7 +66,7 @@ in stdenv.mkDerivation rec {
   '';
 
   meta = with stdenv.lib; {
-    description = "X-Ray Engine 1.6 expansion. Original version was used in S.T.A.L.K.E.R.: Call of Pripyat";
+    description = "Improved version of the X-Ray Engine, the game engine used in the world-famous S.T.A.L.K.E.R. game series by GSC Game World.";
```

Comment:
```suggestion
    description = "Improved version of the X-Ray Engine, the game engine used in the world-famous S.T.A.L.K.E.R. game series by GSC Game World";
```

Reply:
Disagree on the dot. To me, a missing dot is grammatically rude. This is the end of the description and nothing will follow after it. So, it should end with the dot. It is not a funny chat, it is package description!
Also missing dot does not correspond with the project description.

But I can agree with you, if the's a good reason for removing the dot, eg. it looks ugly when you read the description in the terminal or in GUI, or, well, if all the packages don't have the dot in the end of the description...

Reply:
> Disagree on the dot.

Thats the rule. Short descriptions are no sentences and contain no final dot. No arguing.

Reply:
Oops, sorry, I've missed that part of the CONTRIBUTING.md 😄